android/src/main/java/com/swmansion/gesturehandler/core/NativeViewGestureHandler.kt

```diff
@@ -79,6 +86,7 @@ class NativeViewGestureHandler : GestureHandler<NativeViewGestureHandler>() {
       is ReactEditText -> this.hook = EditTextHook(this, view)
       is ReactSwipeRefreshLayout -> this.hook = SwipeRefreshLayoutHook(this, view)
       is ReactScrollView -> this.hook = ScrollViewHook()
+      is ReactTextView -> this.hook = TextHook()
```

Comment:
```suggestion
      is ReactTextView -> this.hook = TextViewHook()
```

Reply:
I didn't like it either 😅 

Changed in [1852e67](https://github.com/software-mansion/react-native-gesture-handler/pull/3202/commits/1852e67f5cf149b59a0f293225c0406b74ff6cc2)

Reply:
You may have forgotten to commit this change or to push it 😅

Reply:
Seems like I did forgot to push it 😅 ([5697078](https://github.com/software-mansion/react-native-gesture-handler/pull/3202/commits/56970785cb8399a87d609e2de3f25086f807b118))